### DISCUSSION TITLE
enumerating non-planar rings substructures

### DISCUSSION THREAD

**Discussion by YoavShamir5** [rdkit/rdkit#6732] [Q&A]
Hello, I would appreciate any advice on the following:

1) Is there a straightforward way to enumerate all non-planar rings that are substructures of an Rdkit molecule?
2) Is there a straightforward way to embed a molecule to generate a single conformer, and then sample multiple conformers only for non-planar ring substructures (keeping the rest of the molecule unchanged), resulting in multiple conformers for the entire molecule at the output? 

Thanks for your help!
Yoav

**Comment by phonglam3103** [comment]
Hi @YoavShamir5. Have u found the answer? If so, please guide me with this.
Thank u.
